README.md

```diff
@@ -11,6 +11,15 @@ The goal is to be **scalable**, **reliable**, fast and extensible.
 We therefore use a _score-based algorithm_ and _no floating-point arithmetic_ whatsoever.
 
 
+## Features
+
+- Supports billions of participants
+- Supports thousands of proposals
```

Comment:
we will need a test to asses it. ideally it would be in CI with a longtest flag. what do you think?

Reply:
Definitely !
I've added a test for the first one.
The second one we'll test outside of the JSON file, because thousands of proposals will make for a big file that will become cumbersome to manage.

Reply:
right. what about a procedural generation of the JSON file?

Reply:
A tip in python to generate a proposal tally, knowing the amount of judgments?   (distribute, knowing the sum?)